tests/app/conftest.py

```diff
@@ -1704,6 +1719,20 @@ def _sample_provider(
     yield _sample_provider
 
     # Teardown
+
+    # Clear template_folder and template_reacted
+    template_ids_query = select(Template.id).where(Template.provider_id.in_(created_provider_ids[worker_id]))
+    template_ids = notify_db_session.session.execute(template_ids_query).scalars().all()
```

Comment:
This is odd. We have not been using execute then casting to scalar. This "smells" and the right way is probably to use the named tuple, calling it template_rows, then `template_row.id` in the for loop.

Reply:
Updated.